src/app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask, request, jsonify
+import joblib
+from libml.preprocessing import get_vectorizer
+
+app = Flask(__name__)
+model = joblib.load("model.joblib")
```

Comment:
This should be a network call to the public URI of the model file so instead of loading it from a local file, we use the model pushed by model-training, wherever it might be.